<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xls_pack</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>751</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Shared keys(seperate by a space)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_k"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>types of the shared keys</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_tk"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Shared useless keys(seperate by a space)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_uk"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>output filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_out"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_match">
        <property name="text">
         <string>match keys with metadata</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="buttondup">
          <property name="text">
           <string>排重</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonmerge">
          <property name="text">
           <string>整合</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
